Formula/git.rb

```diff
@@ -75,6 +76,11 @@ def install
       LDFLAGS=#{ENV.ldflags}
     ]
 
+    # The git-gui and gitk tools are installed by a separate formula (git-gui)
+    # to avoid a dependency on tcl-tk and to avoid using the broken system
+    # tcl-tk (see https://github.com/Homebrew/homebrew-core/issues/36390)
+    args += %w[NO_TCLTK=1]
```

Comment:
Add this to the `args` array above and note the `git-gui`/`gitk` stuff in the caveats.